iphone/Maps/Classes/CustomViews/NavigationDashboard/Views/RoutePreview/RoutePreviewStatus/BaseRoutePreviewStatus.swift

```diff
@@ -84,7 +84,19 @@ final class BaseRoutePreviewStatus: SolidTouchView {
     }
   }
 
+  @objc static func setImagePadding(_ padding: CGFloat, for button: UIButton) {
+    let isRightToLeft = UIView.userInterfaceLayoutDirection(for: button.semanticContentAttribute) == .rightToLeft
```

Comment:
`UIApplication.sharedApplication.userInterfaceLayoutDirection == .rightToLeft` will be enough

Reply:
I don't agree with this one. If we gonna put code this into a global extension, it should be as generic as possible. sharedApplication doesn't care if semanticContentAttribute is explicitly set on some UIView down the ui tree

Reply:
Ok! I accept this argument